screens/Registration/AuthenticationScreen/AuthenticationScreen.tsx

```diff
@@ -143,11 +144,15 @@ const AuthenticationScreen: React.FC = () => {
           value={confirmEmail}
           ref={thirdRef}
         ></InputField>
-        {!keyboardVisible &&
-          <View style={[styles.absolute, {paddingBottom: insets.bottom}]}>
-            <LowBar nextScreen="HomeTabs" buttonText="Confirm" authenticate={submitForm} />
+        {!keyboardVisible && (
+          <View style={[styles.absolute, { paddingBottom: insets.bottom }]}>
+            <LowBar
+              nextScreen="ExploreTabs"
+              buttonText="Confirm"
+              authenticate={submitForm}
+            />
           </View>
-        }
+        )}
       </View>
     </TouchableWithoutFeedback>
   )
```

Comment:
Let's try to have the same code formatter so we don't get these repeated parts of code in the PRs.

Reply:
I will check into it but this is weird that we don't have the same